lib/bloom.js

```diff
@@ -69,7 +69,7 @@ Bloom.prototype.multiCheck = function (topics) {
       t = Buffer.from(t, 'hex')
     }
 
-    match && self.check(t)
+    match = match && self.check(t)
```

Comment:
Why not
```
if (!self.check(t))
  return false
```

Reply:
I tried to reduce the impact on the code:
```
let array4test = [true, false, true];

function testMatch(topics) {
  var match = true;
  topics.forEach(function (t) {
    match = match && t;
  })
  return match;
}

function testReturn(topics) {
  var match = true;
  topics.forEach(function (t) {
    if (!t) return false;
  })

  return match;
}

testMatch(array4test); // =>return false
testReturn(array4test); // =>return true
```

***But if you wont perfomance:***

```
Bloom.prototype.multiCheck = function (topics) {
  var self = this
  return topics.every(function (t) {
    if (!Buffer.isBuffer(t))
      t = Buffer.from(t, 'hex')
    return self.check(t)
  })
}
```